Engine_Revit_UI/Query/BottomLevel.cs

```diff
@@ -41,7 +41,7 @@ public static Level BottomLevel(this oM.Geometry.ICurve curve, Document document
 
             //double aMinElevation = BH.Engine.Geometry.Query.Bounds(curve as dynamic).Min.Z;
             //if (convertUnits)
-            //    aMinElevation = UnitUtils.ConvertToInternalUnits(aMinElevation, DisplayUnitType.DUT_METERS);
+            //    aMinElevation = aMinElevation.FromSI(UnitType.UT_Length);
```

Comment:
If not needed let's remove

Reply:
I am planning to look into this method in 20Q1 - I would keep the commented code until then.

Reply:
Ok